### PR TITLE
Blacklist sansufentanyl from medical allergy quirk

### LINKED ISSUE #76063: Hereditary manifold sickness patients can be allergic to sansufentanyl
## Reproduction:

1. Have allergies
2. Have hereditary manifold sickness

image

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

Fixes https://github.com/tgstation/tgstation/issues/76063. The whole mechanic of having HMS https://github.com/tgstation/tgstation/pull/75035 is that you have 6 pills to live. If you spawn allergic to those pills that's going to be a quick end for you.

## Changelog

:cl: LT3
fix: Sansufentanyl won't roll as a potential extreme allergy for Hereditary Manifold Sickness
/:cl:

### CODE REVIEW

File: code/datums/quirks/negative_quirks.dm [hunk 1]
```diff
@@ -973,7 +973,7 @@
 	quirk_flags = QUIRK_HUMAN_ONLY|QUIRK_PROCESSES
 	mail_goodies = list(/obj/item/reagent_containers/hypospray/medipen) // epinephrine medipen stops allergic reactions
 	var/list/allergies = list()
-	var/list/blacklist = list(/datum/reagent/medicine/c2,/datum/reagent/medicine/epinephrine,/datum/reagent/medicine/adminordrazine,/datum/reagent/medicine/omnizine/godblood,/datum/reagent/medicine/cordiolis_hepatico,/datum/reagent/medicine/synaphydramine,/datum/reagent/medicine/diphenhydramine)
+	var/list/blacklist = list(/datum/reagent/medicine/c2,/datum/reagent/medicine/epinephrine,/datum/reagent/medicine/adminordrazine,/datum/reagent/medicine/omnizine/godblood,/datum/reagent/medicine/cordiolis_hepatico,/datum/reagent/medicine/synaphydramine,/datum/reagent/medicine/diphenhydramine,/datum/reagent/medicine/sansufentanyl)
```
Comment (JohnFulpWillard) [inline code/datums/quirks/negative_quirks.dm:976]: Can you multi-line this please

Comment (lessthnthree) [reply code/datums/quirks/negative_quirks.dm:976]: Sure

**Comment by Sealed101** [comment]
handled by #75654 but that one went stale

**Comment by ZephyrTFA** [comment]
I'm not sure we should give any help to people who are unlucky. They should just get better luck.

**Comment by lessthnthree** [comment]
I am fine with calling it a skill issue.

**Pull request merged**